main/handlersettings.go

```diff
@@ -66,6 +68,16 @@ func (h handlerSettings) validate() error {
 		return errStoragePartialCredentials
 	}
 
+	if (h.protectedSettings.StorageAccountKey != "" || h.protectedSettings.StorageAccountName != "") && !h.protectedSettings.ManagedIdentity.isEmpty() {
```

Comment:
do we need a null check for h.protectedSettings.ManagedIdentity ?

Reply:
Good catch, that was definitely a bug